docs/_pages/contributing.md

```diff
@@ -251,7 +251,41 @@ Discussion about contributing code to Hudi happens on the [dev@ mailing list](co
   * `packaging` : Poms for building out bundles for easier drop in to Spark, Hive, Presto, Utilities
   * `style`  : Code formatting, checkstyle files
 
+## Code WalkThrough
 
+This Quick Video will give a code walkthrough to start with [watch](https://www.youtube.com/watch?v=N2eDfU_rQ_U).
+
+## Docker Setup
+
+We encourage you to test your code on docker cluster please follow this for [docker setup](https://hudi.apache.org/docs/docker_demo.html).
+
+## Remote Debugging 
+
+if your code fails on docker cluster you can remotely debug your code please follow the below steps.
```

Comment:
Can we point out how many steps? e.g. `step 1, step 2...`?

Reply:
yea @yanghua this will give better clarity done changes acc.